api/envoy/admin/v2alpha/server_info.proto

```diff
@@ -17,6 +17,10 @@ message ServerInfo {
     LIVE = 0;
     // Server is draining listeners in response to external health checks failing.
     DRAINING = 1;
+    // Server has not yet started cluster manager initialization.
```

Comment:
The init manager is actually for all things that come after CM init (see cluster_manager_impl and RunHelper is server.cc). Can we rephrase these comments to better indicate the flow? In an optimal world we would also somehow expose CM phase 1 init, phase 2 init, etc. (see ClusterManagerInitHelper but that is more work and can likely be left as a TODO.